uint8 TYPE_CIRCLE=0
uint8 TYPE_RECTANGLE=1
uint8 TYPE_LINE=2

uint8 type
turtlesim/Pose center
float32 radius # used only for circle
float32 width # used for rectangle
float32 height # used for rectangle
float32 length # used for line